src/tests/ie_test_utils/common_test_utils/data_utils.cpp

```diff
@@ -8,11 +8,14 @@
 
 #include <ie_blob.h>
 #include <blob_factory.hpp>
+#include "openvino/core/deprecated.hpp"
 
 using namespace InferenceEngine::details;
 
 namespace CommonTestUtils {
 
+OPENVINO_SUPPRESS_DEPRECATED_START
+
 bool isDenseBlob(const InferenceEngine::Blob::Ptr& blob) {
```

Comment:
to rewrite?

Reply:
Rewriting data_utils.cpp requires a lot of changes. I prefer to do it in next PR.

OPENVINO_SUPPRESS_DEPRECATED_START - is here just for CI passing